package.json

```diff
@@ -12,6 +12,7 @@
     "pretest": "rimraf ./.nyc_output",
     "test": "cross-env NODE_ENV=test nyc mocha --recursive --require @babel/register test",
     "build": "rimraf ./lib && babel src -d lib",
+    "dev": "npm run build -- --watch",
```

Comment:
I'd also be ok with naming this `build:dev` but `dev` is clear and a common convention I've followed in the past that encompasses all appropriate processes to develop the project.